src/components/FormikFileInput/FormikFileInput.tsx

```diff
@@ -1,17 +1,26 @@
 import { pulseAnimation } from "@/styles/animations"
-import { Typography } from "@mui/material"
+import { Grid, GridProps } from "@mui/material"
+import Chip from "@mui/material/Chip/Chip"
+import Typography from "@mui/material/Typography/Typography"
 import { useField } from "formik"
 import { ReactElement, useState } from "react"
 import { useTranslation } from "react-i18next"
 import styled, { css } from "styled-components"
 
-interface FormikFileInputProps {
+interface FormikFileInputProps extends Pick<GridProps, "sx"> {
```

Comment:
Instead of picking up sx prop form GridProps just use SxProps coming from MuI

import { SxProps, Theme } from '@mui/material/styles';

interface ListHeaderProps {
  children: React.ReactNode;
  sx?: SxProps<Theme>;
}